docs/src/interface_fluxes.md

```diff
@@ -0,0 +1,103 @@
+# Interface fluxes
+
+`ClimaOcean`'s `OceanSeaIceModel` has essentially two goals:
+    1. Manage time-stepping multiple component models forward simulatneously,
+    2. Compute and/or pass fluxes between the component models.
```

Comment:
@glwagner, this is not proper markdown formatting and it does not appear as you intended